### ISSUE TITLE
Extraordinarily large inter_module folder.

### ISSUE THREAD

**Issue by martedva** [CSTR-Edinburgh/merlin#325]
Hello,

We are creating an enormous AVM with the entire VTCK corpus, i.e. 41,267 utterances. In this process we have successfully trained the duration and acoustic model. 

The duration model came out at a reasonable size of 7.2GB, however, the acoustic model takes up 146.5GB. The culprit seems to be the inter_module folder of the acoustic model, which takes up 134.7GB.

The contents of the folder is shown below:

![image](https://user-images.githubusercontent.com/6831315/38165084-5d87f0b6-350e-11e8-94ae-c56afa58e570.png)

Is this a reasonable size for the folders shown, or am I missing some kind of clean-up from one of the scripts?

Thanks in advance!

**Comment by mdekorte** [comment]
Yes, that seems quite reasonable. Training on 1500 sentences takes up approximately 6GB of space for me, so if you do the math, you could easily end up with 135GB. The main culprits are the acoustic features that are extracted for the vocoder and then stored, they take up considerable space. 

However, most of the inter_module directories are only stored in case you ever want to retrain your model or if you want to calculate objective measures (Please correct me if I forget to mention other purposes.). If you don’t wanna do either, you can decide to clean up the inter_module directory. Or store them somewhere else, in case you ever want to train a new model (say, RNN style instead of DNN) and don’t want to spend a lot of time waiting for the MAKECMP step to finish. Then just set NORMLAB, MAKECMP and NORMCMP to False in your config file and specify where the features are stored in your config file (at the inter_data line). Then the whole training process will be much faster when you want to train a model, since the acoustic features don’t have to be regenerated. :)
